### PR TITLE
Template changes

### LINKED ISSUE #36: Incorrect Grammar on the "contribute to this guide" page
The sentence on the "contribute to this guide" page uses incorrect grammar - "Is something missing or needs to be fixed?" The subject is "something" and the verb is "is" which makes it "something is missing" and "something is needs" - should be "needing". This is used throughout interactive and static guides.

OpenLiberty/iguides-common#98 is the same

### PR DESCRIPTION
**Pull request by Kubik42**
Tentative template changes for #28. Also fixes #36

### CODE REVIEW

**Comment by Kubik42** [comment]
Are we good to pull these into the master branch @evelinec @lauracowen @yeekangc ?

**Comment by yeekangc** [comment]
@bcpswope or a member of her team should review and approve.

Good to give the broader team a summary of the changes when ready too.

**Comment by evelinec** [comment]
@bcpswope We went through these changes with you and Brandy in the meeting(s) we had. Can you take a look at this PR, and let us know any further feedback you may have? Thank you.

File: template.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@
 = Title of guide, starting with a gerund (Creating, Administering, and so on)
 
 // Start the introduction with "You'll explore how to..." or something similarly catchy:
-You'll explore how to....
+Learn how to....
 // Write up to two sentences and fifty words, with meaningful information on what the user can accomplish with this guide.
```
Comment (bcpswope) [inline template.adoc]: An ellipsis is a set of three periods ( . . . ) indicating an omission.  Looks like we have an extra one.

Comment (bcpswope) [inline template.adoc]: Recommend: "Write no more than two sentences, or about fifty words, with. . ."

File: template.adoc [hunk 2]
```diff
@@ -33,7 +33,7 @@
 = Title of guide, starting with a gerund (Creating, Administering, and so on)
 
 // Start the introduction with "You'll explore how to..." or something similarly catchy:
-You'll explore how to....
+Learn how to....
```
Comment (bcpswope) [inline template.adoc]: Nice!

File: template.adoc [hunk 3]
```diff
@@ -49,21 +49,34 @@ You'll explore how to....
...
 // Add this getting started section to your guide if it is applicable. Use the following include to pull in the git clone instructions from the guides-common repo.
-
+== Getting started
 include::{common-includes}/gitclone.adoc[]
 
+// This is a subsection of the "Getting started" section above. It should briefly walk the user through
+// how to setup everything in the "finish" directory and try out the finished version of what
+// they will be building.
+=== Try what you'll build
+
```
Comment (bcpswope) [inline template.adoc]: I still think Try what you'll build could be improved:
Try it out? View the end result? See what you will build? See it first? I know we were talking about this in our first call, not sure what was decided.

File: template.adoc [hunk 4]
```diff
@@ -49,21 +49,34 @@ You'll explore how to....
...
 
-== Building the application
-// Add this Building the application section to your guide if it is applicable.
+// Add this section to your guide if it is applicable.
 // Put this section in the appropriate order in your guide.
-// Use the following include to pull in the Maven build instructions from the guides-common repo.
+== Building and running the application
 
+// Use the following include to pull in the Maven build instructions from the guides-common repo.
 include::{common-includes}/mvnbuild.adoc[]
```
Comment (bcpswope) [inline template.adoc]: Use the following "include"?  I think using include as a noun may cause some confusion. Use the following example? code? text? command?

File: template.adoc [hunk 5]
```diff
@@ -76,15 +89,24 @@ include::{common-includes}/mvnbuild.adoc[]
 // Use tick marks around directories, files, values, class names, method names, and so on.
```
Comment (bcpswope) [inline template.adoc]: Do we give an example for them?

For example, a directory can be coded: ```open-liberty\dev\cnf\release\dev\openliberty\<version>\openliberty-xxx.zip```.

File: template.adoc (hunk 3 above)
Comment (Kubik42) [reply template.adoc]: Personally, I think "Try what you'll build" makes the most sense. Its broad enough to include everything in the guide, but not too broad to create confusing. Its also not specific enough to single out parts of the guide.

**Comment by Kubik42** [comment]
Thank you Brandi! I made tweaks to reflect your comments. Merging.

**Pull request merged**